asreview/analysis/readers.py

```diff
@@ -0,0 +1,21 @@
+import os
+
+from asreview.logging.utils import Logger
+
+
+def loggers_from_dir(data_dir, prefix="result"):
```

Comment:
Move this function to `logging.utils`. Pretty useful function. Please extend docstring and sphinx reference.